src/base.ts

```diff
@@ -1,12 +1,19 @@
-import { Reshuffle, BaseHttpConnector } from 'reshuffle-base-connector'
+import { Reshuffle, BaseHttpConnector, EventConfiguration } from 'reshuffle-base-connector'
 import { Client, ClientOptions } from '@microsoft/microsoft-graph-client'
+import type MicrosoftGraph from '@microsoft/microsoft-graph-types'
+import type { Request, Response } from 'express'
 
 import { ClientCredentialAuthenticationProvider, MicrosoftOptions } from './auth'
 
 export type MicrosoftConnectorConfigOptions = MicrosoftOptions & { debugLogging?: boolean }
+
+export type MicrosoftEventConfigOptions = MicrosoftGraph.Subscription
+
+const DEFAULT_WEBHOOK_PATH = '/reshuffle-microsoft-connector/webhook'
```

Comment:
In both Monday and quickbooks we added a convention of `/webhooks/xxx` where `xxx` is the service name. I think we should stick to this?